util/proactor_base.cc

```diff
@@ -66,6 +66,8 @@ ProactorBase::ProactorBase() : task_queue_(512) {
 }
 
 ProactorBase::~ProactorBase() {
+  on_idle_arr_.clear();
+
```

Comment:
Doesn't this happen on destruction either way 😅

Reply:
yeah, I guess you are right.